include/mutex.h

```diff
@@ -69,11 +69,6 @@ void mtx_unlock(mtx_t *m);
 
 DEFINE_CLEANUP_FUNCTION(mtx_t *, mtx_unlock);
 
-/*! \brief Mutex owner getter. */
-static inline volatile thread_t *mtx_owner(mtx_t *m) {
-  return m->m_owner;
-}
-
```

Comment:
@cahirwpz weren't we supposed to replace `->m_owner` in tests with a getter?

Reply:
(In our turnstile tests.)

Reply:
Ah... ok. I'm going to restore this function then.